slice/IceGrid/Admin.ice

```diff
@@ -130,6 +130,9 @@ module IceGrid
 
         /// The path to the node data directory.
         string dataDir;
+
+        /// The Ice SO version used by this node, for example "38". An empty string means this version is unknown.
+        string iceSoVersion;
```

Comment:
Doesn't this break compatibility with older versions?

Reply:
You're right. I thought we did not maintain interface compatibility for IceGrid, but we actually do:
https://doc.zeroc.com/ice/3.7/release-notes/backward-compatibility-of-ice-versions#id-.BackwardCompatibilityofIceVersionsv3.7-IceGrid

It's only for the IceGrid admin tool that we don't maintain compatibility.

Reply:
Fixed.